spec/menu_spec.rb

```diff
@@ -0,0 +1,34 @@
+require 'menu'
+
+describe Menu do
+
+before(:each) do
+  subject.add_dish("Tomatoes mozarella",10)
+end
+
+  it 'allows to add dishes to the list with name and price' do
+    expect(subject).to respond_to(:add_dish).with(2).arguments
+  end
+
+  it 'allows to display dishes as a well formatted list' do
+    subject.add_dish("Prosciutto",12)
+    expect(subject.display_menu).to eq("Tomatoes mozarella: £10, Prosciutto: £12")
+    end
```

Comment:
Lint/BlockAlignment: `end` at 16, 4 is not aligned with `it 'allows to display dishes as a well formatted list' do` at 13, 2.